public/static/locales/cy/header.json

```diff
@@ -9,5 +9,7 @@
   "listings": "Rhestrau",
   "home": "Hafan",
   "travel": "Cyfeiriadur Teithio",
-  "mobile": "Yn ôl i'r hafan"
+  "mobile": "Yn ôl i'r hafan",
+  "bannerHeading": "Rydyn yn symud i HelpwrArian",
+  "banner": "Bydd Pension Wise yn rhan o HelpwrArian, y ffordd hawdd o gael cymorth ar gyfer eich arian a phensiynau."
```

Comment:
Can we check the copy here? This appears to be the copy for Pension Wise where (I guess) we need the copy for MAS.

Reply:
Fixed